LBBT-Design-Patterns/LBBTCalculator.cs

```diff
@@ -1,60 +1,66 @@
-﻿using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace LBBT_Design_Patterns
+﻿using System.Collections.Concurrent;
+
+namespace LBBT_Design_Patterns;
+
+/// <summary>
+///   Tax calculator using tax bands with rates.
+/// </summary>
+/// <param name="taxStrategies">
+///   Tax strategies to use in calculations,
+///   or will use default strategies specified in <see cref="CreateDefaultStrategies"/> if null.
+/// </param>
+public class LBBTCalculator(IList<ITaxStrategy>? taxStrategies = null) : ITaxCalculator
 {
-    internal class LBBTCalculator
+    private readonly IList<ITaxStrategy> _taxBands = taxStrategies ?? CreateDefaultStrategies();
```

Comment:
If a list of tax strategies wasn't provided then "taxStrategies" will be null and therefore we will use the return value of CreateDefaultStrategies() instead. Look up "c# null coalescing" for an explanation.